dspace-server-webapp/src/test/java/org/dspace/app/rest/RegistrationRestControllerIT.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import java.util.Iterator;
+import java.util.List;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.lang3.StringUtils;
+import org.dspace.app.rest.model.RegistrationRest;
+import org.dspace.app.rest.test.AbstractControllerIntegrationTest;
+import org.dspace.eperson.RegistrationData;
+import org.dspace.eperson.dao.RegistrationDataDAO;
+import org.dspace.services.ConfigurationService;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class RegistrationRestControllerIT extends AbstractControllerIntegrationTest {
```

Comment:
This IT class can be removed. The `RegistrationRestController` was removed and refactored into `RegistrationRestRepository`.  So, these tests can also be moved if they don't already exist elsewhere or simply removed if they are duplicative.

Keep in mind, if the tests here are kept, they need cleanup. They are not cleaning up after a POST properly (in both methods) and the `forgotPasswordTest()` test is strangely calling `context.complete()` and turning off authorization around a simple config change.

I *think* these same functions are already tested elsewhere though, so I'd lean towards removing this IT class entirely as it looks duplicative.

Reply:
The tests have been moved, they were not duplicates
The test data is cleaned up, but as noted in https://github.com/DSpace/DSpace/pull/2763#discussion_r445574105 the alternative mode for cleanup can't be used here